src/server/services/about/transformers/service-status.js

```diff
@@ -9,7 +9,7 @@ const githubOrg = config.get('githubOrg')
 
 function serviceStatus(service) {
   const createStatusInfo = service.serviceStatus
-  const createRepository = createStatusInfo?.createRepository
+  const createRepository = createStatusInfo?.['cdp-create-workflows']
```

Comment:
Is this called the right thing now? `createRepository` ?

Reply:
`createRepository` is no more, its handled the same as all of the other workflows hence its status field its expecting to receive status updates from.